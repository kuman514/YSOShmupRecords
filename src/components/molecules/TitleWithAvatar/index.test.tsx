import React from 'react';
import { cleanup, render } from '@testing-library/react';
import { beforeEach, describe, expect, it } from 'vitest';
import 'jest-styled-components';

import { TitleWithAvatar } from '.';

describe('TitleWithAvatar', () => {
  beforeEach(() => {
    cleanup();
  });

  it('should have a snapshot match for its appearance', () => {
    const { container } = render(<TitleWithAvatar />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
