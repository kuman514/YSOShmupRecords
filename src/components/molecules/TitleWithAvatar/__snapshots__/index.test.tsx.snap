// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`TitleWithAvatar > should have a snapshot match for its appearance 1`] = `
.c2 {
  display: flex;
  justify-content: center;
  align-items: center;
  border-radius: calc(150px / 2);
  width: 150px;
  height: 150px;
  background-color: var(--gray-color);
}

.c3 {
  color: #000000;
  font-family: Inter;
  font-size: 16px;
  font-weight: 500;
  border-radius: inherit;
  width: inherit;
  height: inherit;
}

.c0 {
  height: 100%;
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  row-gap: 36px;
}

.c1 {
  width: 30px;
  border-top: 1px solid #ffffff;
}

.c4 {
  color: #ffffff;
  font-family: Inter;
  font-size: 36px;
  font-weight: 200;
}

<div
  class="c0"
>
  <div
    class="c1"
  />
  <div
    class="c2"
  >
    <img
      alt="YSOShmupRecords avatar"
      class="c3"
      src="/src/assets/avatar/avatar.jpeg"
    />
  </div>
  <h1
    class="c4"
  >
    YSOShmupRecords
  </h1>
</div>
`;
